twilight-model/src/http/interaction.rs

```diff
@@ -102,6 +103,11 @@ pub enum InteractionResponseType {
     Modal = 9,
     /// Respond to an interaction with an upgrade button, only available
     /// for apps with monetization enabled
+    ///
+    /// Please send a [`InteractionResponseType::ChannelMessageWithSource`]
+    /// with an [`Button`](crate::channel::message::component::Button) with the style [`ButtonStyle::Premium`](crate::channel::message::component::ButtonStyle)
+    /// instead.
+    #[deprecated(note = "This response type has been deprecated by Discord.")]
```

Comment:
A note like "Deprecated by Discord in favor of Premium Buttons" would be more informative